cluster/addons/pod-resizer/src/main/pod_nanny.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"flag"
```

Comment:
Our other code uses `"github.com/spf13/pflag"` instead of the built-in flag package.